bridge/client/app/_pipes/to-type.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
so much cleaner with a pipe than the cast-methods, love it 👌

Reply:
Yep, that's much better :)